clients/sfizz_jack.man.in

```diff
@@ -1,31 +1,34 @@
-.TH sfizz_jack 1 "@MAN_TODAY@" "@CMAKE_PROJECT_VERSION@" "sfizz_jack man page"
+.TH sfizz_jack 1 "@MAN_TODAY@" "@CMAKE_PROJECT_VERSION@" "sfizz"
```

Comment:
I'm clueless about man pages. This changes the title of the page?

Reply:
It changes the section title (the page title remains  “sfizz_jack”). It used to say it belonged to the “sfizz_jack man page” section, which made little sense; the patch changes it to say it belongs to the “sfizz” section.